Chinese/ZZB/zzb.md

```diff
@@ -1,3 +1,6 @@
 至尊宝：谁说我斗鸡眼？我只是把视力集中在一点，以改变我以往对事物的看法！
 
-If you speak English, [go here](../../english/marshmallow.md).
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.